centreon/src/Security/WebSSOAuthenticator.php

```diff
@@ -180,7 +187,7 @@ public function authenticate(Request $request): SelfValidatingPassport
                 ),
                 200
             );
-        } catch (SSOAuthenticationException $exception) {
+        } catch (SSOAuthenticationException $exception) { // @todo: what then if a different exception?
```

Comment:
```suggestion
        } catch (SSOAuthenticationException $exception) {
```
Le système d'authentification de Symfony va capturer d'autres exceptions.
Le but ici est de capturer l'exception pour modifier son message d'erreur.